jscript.js

```diff
@@ -313,4 +313,84 @@ clickbtn.forEach((button) => {
       body.removeChild(pupmain);
     });
   });
-});
\ No newline at end of file
+});
+
+// form validation feature
+const form = document.querySelector('form');
+const email = document.getElementById('mail');
+const nameval = document.getElementById('name');
+const messageval = document.getElementById('message');
+const emailError = document.querySelector('.erroremail');
+const nameerror = document.querySelector('.errorname');
+const messageerror = document.querySelector('.errormessage');
+
+function showError() {
+  // name validation
+  if (nameval.validity.valueMissing) {
+    nameerror.innerHTML = 'Name filed can"t be empty!!!.';
+    nameerror.className = 'error';
+  } else if (nameval.value.match('[0-1]+')) {
+    nameerror.innerHTML = 'Name can not be have numeric !!!';
+    nameerror.className = 'error';
+  } else {
+    nameerror.innerHTML = '';
+    nameerror.className = 'success';
+  }
+  // email validation
+  if (email.validity.valueMissing) {
+    emailError.innerHTML = 'Email filed can"t be empty!!!.';
+    emailError.className = 'error';
+  } else if (email.validity.typeMismatch) {
+    emailError.innerHTML = 'Entered value needs to be an email address !!!.';
+    emailError.className = 'error';
+  } else if (email.value.match('[A-Z]+')) {
+    emailError.innerHTML = 'It is not valide email, all letter should be in lower case !!!.';
+    emailError.className = 'error';
+  } else {
+    emailError.innerHTML = '';
+    emailError.className = 'success';
+  }
+  // validate message
+  if (messageval.validity.valueMissing) {
+    messageerror.innerHTML = 'Message filed can"t be empty!!!.';
+    messageerror.className = 'error';
```

Comment:
- [ ] You did a great job 😃 at adding the functionality of the validation of the contact if in lowercase, but I see when I write valid information for the contact and try 💯 to submit it is not submitting it, so please try to fix that to let the users communicate with you 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| My Form | Your Form| 
| --- | --- |
|  ![Recording 2023-04-19 at 12 28 05 (1)](https://user-images.githubusercontent.com/83360636/233047604-36c8e17a-4bdd-4b30-b2c4-8aa0f35e1c5f.gif)  |  ![Recording 2023-04-19 at 12 26 33](https://user-images.githubusercontent.com/83360636/233047188-a6d9cb0f-c1b2-485a-9d9e-e74307e01714.gif)  |

Reply:
## Okay Thank you I will correct it.